presto-product-tests/conf/docker/common/jdbc_db.yml

```diff
@@ -24,3 +24,12 @@ services:
       MYSQL_PASSWORD: swarm
       MYSQL_ROOT_PASSWORD: swarm
       MYSQL_DATABASE: test
+
+  sqlserver:
+    hostname: sqlserver
+    image: 'microsoft/mssql-server-linux'
+    ports:
+      - '1433:1433'
```

Comment:
I have created this issue to track this for SQL Server in Docker https://github.com/Microsoft/mssql-docker/issues/21